ch.theologeek.Manuskript.appdata.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be maintained upstream, ideally, it's not specific to Flathub.

Reply:
I definitely have those files in the Manuskript repository as well. So in future versions I can get them from the archive. However in version 0.12.0 they aren't part of the archive yet.